packages/flutter/test/widgets/async_test.dart

```diff
@@ -118,6 +118,25 @@ void main() {
       await eventFiring(tester);
       expect(find.text('AsyncSnapshot(ConnectionState.done, null, bad)'), findsOneWidget);
     });
+    testWidgets('runs the builder using given initial data', (WidgetTester tester) async {
+      final GlobalKey key = new GlobalKey();
+      await tester.pumpWidget(new FutureBuilder<String>(
+        key: key, future: null, builder: snapshotText, initialData: 'I'
```

Comment:
Here and similar places below, I think we'd want both one line per parameter and a trailing comma. Either we do
```dart
 aFunctionCallThatFitsOnOneLine(some: key, no: trailing, comma: see);
```
or we do
```dart
 aFunctionCallThatNeedsMultipleLines(
   because: we,
   have: so,
   many: parameters,
   andWeThenNeed: a,
   trailing: comma,
 );
```
The point of doing the above is that 1) it obeys the Flutter style guide, and 2) `dartfmt` leaves it unchanged.

https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#use-a-trailing-comma-for-arguments-parameters-and-list-items-but-only-if-they-each-have-their-own-line

Reply:
Yah no worries... that's how I normally do it, but I wasn't 100% sure about the Flutter style b/c this file is full of lines like that:

https://github.com/flutter/flutter/pull/13820/files#diff-ecef29ade55b943f96ea50b77461443fR88

Should I file a follow-up issue to fix the styling for this file?

Reply:
I see your point. We'll leave it for now then. I believe Flutter style will eventually align with `dartfmt`, and we should generally stay as close to `dartfmt`'s output as possible without violating the Flutter style guide. I'll merge this PR.

Reply:
This particular file isn't yet in strict adherence to either dartfmt or the Flutter style guide, unfortunately.

Reply:
(I don't mean the changes from this PR, I mean the file as it was before this PR.)

Reply:
What's wrong? (Apart from an very long line towards the end?)

Reply:
Oh, my bad. I had meant the same lines you were commenting on here, but I see now that those were just in this patch. I got confused by the "View" option on GitHub (which shows the new file but without the diff markings).